<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordQueryPage</class>
 <widget class="QWidget" name="RecordQueryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecordQueryPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RecordQueryPage{
	border-image: url(:/MBRClient/Resources/image/r_bg.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:16px;
}
#btn_refresh{
	border-image: url(:/MBRClient/Resources/image/query/query_refresh.png);
}
#btn_refresh:hover{
	border-image: url(:/MBRClient/Resources/image/query/query_authq_hover.png);
}
#btn_refresh:pressed{
	border-image: url(:/MBRClient/Resources/image/query/query_refresh_pressed.png);
}
#btn_refresh:disabled{
	border-image: url(:/MBRClient/Resources/image/query/query_refresh_disabled.png);
}
#btn_delete{
	border-image: url(:/MBRClient/Resources/image/query/delete_record.png);
}
#btn_delete:hover{
	border-image: url(:/MBRClient/Resources/image/query/delete_record_hover.png);
}
#btn_delete:disabled{
	border-image: url(:/MBRClient/Resources/image/query/delete_record_disabled.png);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_enum">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_rnum">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_refresh">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>132</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_top" native="true">
     <property name="styleSheet">
      <string notr="true">#btn_previous{
	border-image: url(:/MBRClient/Resources/image/query/query_pre.png);
}
#btn_previous:hover{
	border-image: url(:/MBRClient/Resources/image/query/query_pre_hover.png);
}
#btn_previous:pressed{
	border-image: url(:/MBRClient/Resources/image/query/query_pre_pressed.png);
}
#btn_previous:disabled{
	border-image: url(:/MBRClient/Resources/image/query/query_pre_disabled.png);
}
#btn_next{
	border-image: url(:/MBRClient/Resources/image/query/query_next.png);
}
#btn_next:hover{
	border-image: url(:/MBRClient/Resources/image/query/query_next_hover.png);
}
#btn_next:pressed{
	border-image: url(:/MBRClient/Resources/image/query/query_next_pressed.png);
}
#btn_next:disabled{
	border-image: url(:/MBRClient/Resources/image/query/query_next_disabled.png);
}
#lbl_ct{
font-size:13px;
font:bold;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_ct">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_previous">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_next">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgt_bottom" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>153</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>153</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="PictureShowWidget" name="wgt_afo" native="true">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>151</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PictureShowWidget" name="wgt_est" native="true">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>151</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>394</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PictureShowWidget</class>
   <extends>QWidget</extends>
   <header>PictureShowWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MBRClient.qrc"/>
 </resources>
 <connections/>
</ui>
